From e4290403a85c1e5f691f50c0eff0417138ef4965 Mon Sep 17 00:00:00 2001
From: Richard Levitte <levitte@openssl.org>
Date: Thu, 23 May 2024 08:04:42 +0200
Subject: [PATCH 111/228] VMS: Redefine _XOPEN_SOURCE_EXTENDED with the value 1
 in apps/ocsp.c

Some versions if the VMS C system header files seem to require this.

Fixes #24466 on release older than 3.3.

Reviewed-by: Tomas Mraz <tomas@openssl.org>
Reviewed-by: Todd Short <todd.short@me.com>
(Merged from https://github.com/openssl/openssl/pull/24471)

(cherry picked from commit 54e9e254feefb153b4b94330f9d3f20b6120c106)
---
 apps/ocsp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/apps/ocsp.c b/apps/ocsp.c
index 821e224c6c..73841750ec 100644
--- a/apps/ocsp.c
+++ b/apps/ocsp.c
@@ -11,7 +11,7 @@
 
 #ifdef OPENSSL_SYS_VMS
   /* So fd_set and friends get properly defined on OpenVMS */
-# define _XOPEN_SOURCE_EXTENDED
+# define _XOPEN_SOURCE_EXTENDED 1
 #endif
 
 #include <stdio.h>
-- 
2.39.5 (Apple Git-154)

